templates/Web/Pages/Angular.MasterDetail/src/app/app-shell/Param_SourceName_Kebab/master-detail.service.ts

```diff
@@ -19,13 +19,13 @@ export class MasterDetailService {
 }
 
 export interface IMasterDetailText {
-  paragraph: string;
   title: string;
   tabName: string;
   id: number;
   status: string;
-  orderDate: Date;
+  orderDate: string;
```

Comment:
why the change of orderDate to string? Just inquiring.

Reply:
cause when they use order Date, it was orderDdate  = new Date().**toDateString**; thus it is a string actually